language/e2e-tests/src/common_transactions.rs

```diff
@@ -24,6 +25,24 @@ lazy_static! {
     pub static ref ROTATE_KEY: Vec<u8> = { rotate_key() };
 }
 
+/// Returns a transaction to add a new validator
+pub fn add_validator_txn(
+    sender: &Account,
+    new_validator: &Account,
+    seq_num: u64,
+) -> SignedTransaction {
+    let mut args: Vec<TransactionArgument> = Vec::new();
+    args.push(TransactionArgument::Address(*new_validator.address()));
+
+    sender.create_signed_txn_with_args(
+        ADD_VALIDATOR_TXN.clone(),
```

Comment:
this looks like a different pattern than CREATE_ACCOUNT? probably worth unifying in one way or another

Reply:
Yeah, in writing this code I noticed that there's a lot of boilerplate required in adding new transaction scripts. Planning to consult with @runtian-zhou in figuring out where we can streamline things a bit.